packages/gestalt/src/Popover/usePopover.ts

```diff
@@ -74,13 +71,20 @@ export default function usePopover({
   caretElement,
   caretPadding,
   direction,
+  forceDirection,
   strategy,
   scrollBoundary,
   hideWhenReferenceHidden,
   onPositioned,
 }: Props): UseFloatingReturn {
-  const isForceDown = direction === 'forceDown';
-  const placement = isForceDown ? 'bottom' : direction;
+  let placement: Placement = direction ?? 'bottom';
+  const isRtl = typeof document === 'undefined' ? false : document?.dir === 'rtl';
+
+  if (isRtl && direction === 'left') {
```

Comment:
@AlbertCarreras have you tested this bahvior? I though @floating-ui library already handled RTL. 🤔

Reply:
It might flip accordingly... but if we force behavior idk... I'll test it anyways.

Reply:
just tested it... no, when forced... it stays i the right/left, cos they don't do end/start and we read LRT/LTR from the document